README.rst

```diff
@@ -1,3 +1,16 @@
+Project archived. No maintenance anymore.
+=========================================
+
+This project is not maintained anymore and is archived. All the current Open-Needs IDE language features
+are moved into `Sphinx-Needs <https://github.com/useblocks/sphinxcontrib-needs>`_. The features will be
+automatically detected and integrated into `Esbonio language server <https://github.com/swyddfa/esbonio>`_.
```

Comment:
I would also just refer to reStructuredText-Pluin in Vs-Code and tell that this detects all features automatically if Sphinx-Needs is used inside a projects.

That's it. No technical details, no esbonio, no thanks.

Maybe a hint,  that further development will be done inside the Sphinx-Needs projects.